app/controllers/welcome_controller.rb

```diff
@@ -1,13 +1,9 @@
 class WelcomeController < ApplicationController
-  # https://github.com/heartcombo/devise#controller-filters-and-helpers
-  before_action :authenticate_user!, only: [:protected]
-  # Alternative: before_action :authenticate_user!, :except => [:index]
-
   def index
     # Welcome page, accessible without login
   end
 
-  def protected
+  def login
     # Only accessible by logged in users, see `before_action` call
```

Comment:
This comment seems to be outdated if the above section about the `before_action` call got deleted, doesn't it?

Overall what do the changes in this file mean in the context of the issue? 🤔